tests/scf-cholesky-basis/input.dat

```diff
@@ -0,0 +1,159 @@
+# Reference energies
+e_cholesky_4  = -230.64594709545503
+e_cholesky_5  = -230.64614462290461
+e_cholesky_6  = -230.64617987370553
+e_cholesky_7  = -230.64620544189239
+e_cholesky_8  = -230.64620513216849
+e_cholesky_9  = -230.64620507251573
+e_cholesky_10 = -230.64620507295609
+e_cholesky_11 = -230.64620504920356
+e_canonical   = -230.64620504921135
```

Comment:
This is actually quite interesting - but not surprising: the energy goes down going down from large thresholds at the beginning, but then goes up starting from the 1e-7 threshold.